examples/server.c

```diff
@@ -108,7 +108,9 @@ int main(int argc, char** argv) {
     signal(SIGINT, stopHandler); /* catches ctrl-c */
 
     UA_ServerNetworkLayer nl = UA_ServerNetworkLayerTCP(UA_ConnectionConfig_standard, 16664);
-    UA_ServerConfig config = UA_ServerConfig_standard;
+    UA_ServerConfig config;
+    UA_ServerConfig_standard_new(&config);
```

Comment:
Should i modify the server and everything else to use a pointer to the config then, instead of copying it around? This would require a lot of minor changes (mainly changing . to ->) i believe.
We could still copy the malloced struct into the one in the server but that might break things later on, right?